app/command_test.go

```diff
@@ -179,12 +179,15 @@ func TestHandleCommandResponsePost(t *testing.T) {
 	assert.Equal(t, resp.IconURL, post.Props["override_icon_url"])
 	assert.Equal(t, "true", post.Props["from_webhook"])
 
+	// Not a built In command
```

Comment:
It's not super clear to me why we test this one as a non-built-in command, but all the previous ones as such. Can you elaborate?

Reply:
it is not clear to me as well, but I understand the built-in command is a native command, and in the other tests we are testing a slack parse which I think is not a native command, right? built-in commands we might not have slack things to parse instead we use mattermost defaults objects. I can be completely wrong in that.

Reply:
Looking at `HandleCommandResponsePost`, it doesn't look like we use the `builtIn` flag to decide whether or not to process slack text. So I'd propose we leave this bit set as previously.

Reply:
done